### PR TITLE
Cache improvements

### PR DESCRIPTION
**Pull request by michaelarnauts**

### CODE REVIEW

File: resources/lib/vtmgo/vtmgo.py [hunk 1]
```diff
@@ -363,7 +367,7 @@ def get_swimlane(self, swimlane=None):
         items = []
         for item in result.get('teasers'):
             if item.get('target', {}).get('type') == self.CONTENT_TYPE_MOVIE:
-                movie = self.get_movie(item.get('target', {}).get('id'), cache=True)
+                movie = self.get_movie(item.get('target', {}).get('id'), cache=CACHE_ONLY)
```
Comment (dagwieers) [inline resources/lib/vtmgo/vtmgo.py]: I assume CACHE_ONLY means that it does not update the cache when expired?
But still downloads if the cache is missing?

Comment (dagwieers) [reply resources/lib/vtmgo/vtmgo.py]: Hmm, it seems it returns None if there is no cache.

Comment (michaelarnauts) [reply resources/lib/vtmgo/vtmgo.py]: CACHE_ONLY is to use only the cache, and not make an API call.

This is used to give extra data when a big listing is requested and I don't want to delay the listing itself. Normally, everything should be already in the cache, but in the odd case it isn't (disabled by user, cache is still downloading), I don't want to have the user wait a few minutes for the listing to appear.

Comment (michaelarnauts) [reply resources/lib/vtmgo/vtmgo.py]: Cache doesn't expire at the moment IIRC, but when you open a program, I always download fresh data, and the cache is updated.

**Pull request merged**